public/pages/CreateTrigger/containers/CreateTrigger/CreateTrigger.js

```diff
@@ -121,17 +122,24 @@ export default class CreateTrigger extends Component {
   onRunExecute = (triggers = []) => {
     const { httpClient, monitor, notifications } = this.props;
     const formikValues = monitorToFormik(monitor);
+    const searchType = formikValues.searchType;
     const monitorToExecute = _.cloneDeep(monitor);
     _.set(monitorToExecute, 'triggers', triggers);
-    if (formikValues.searchType !== SEARCH_TYPE.AD) {
+
+    if (searchType === SEARCH_TYPE.QUERY || searchType === SEARCH_TYPE.GRAPH) {
       const searchRequest = buildSearchRequest(formikValues);
       _.set(monitorToExecute, 'inputs[0].search', searchRequest);
     }
+    if (searchType === SEARCH_TYPE.LOCAL_URI) {
+      const localUriRequest = buildLocalUriRequest(formikValues);
+      _.set(monitorToExecute, 'inputs[0].uri', localUriRequest);
+    }
+
```

Comment:
Make this an `else if` and add an else statement that would log information stating an unsupported `searchType` was found